bundle.yaml.j2

```
@@ -5,6 +5,16 @@ name: cos-lite
 description: >
   COS Lite is a light-weight, highly-integrated, observability stack running on Kubernetes
 applications:
+  traefik:
+    charm: {{ traefik|default('traefik-k8s', true) }}
+    scale: 1
+    trust: true
+    {%- if traefik is defined and traefik.endswith('.charm') %}
+    resources:
+        traefik-image: "jnsgruk/traefik"
```

```suggestion
        traefik-image: "jnsgruk/traefik:2.6.1"
```

Might be good to pin the version of Traefik here?

---

It’s only for the local tests in which we want to build traefik from source. In reality, I think the logic that uses the traefik version should use the upstream-source tag instead. No matter _what_ we put here, will eventually be wrong.